docs/_documentations/mdt-intellij-getting-started.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+title: Installing Codewind on IntelliJ
+description: Installing Codewind on IntelliJ
+keywords: install, installing, IntelliJ, Eclipse, Codewind, IDE, plugin, plug-in, settings, creating, project, projects, template, code change, edit, edits, application, removing
+duration: 1 minute
+permalink: mdt-intellij-getting-started
+type: document
+---
+
+# Getting started with Codewind on IntelliJ
+Use Codewind on IntelliJ to develop your applications.
+
+## Prerequisites
+- Install [IntelliJ](https://www.jetbrains.com/idea/download/#section=mac).
+- Install Docker.
+- If you use Linux, also install Docker Compose.
+
+## Installing Codewind on IntelliJ
+1. Go to the [Eclipse Downloads page](https://download.eclipse.org/codewind/?d) for Codewind. Click **milestone** and **0.9.0**. Then, click **codewind-intellij-0.9.0.zip** to download the file.
```

Comment:
Note: This file path currently doesn't exist, but we speculate a similar file path will be in place with the 0.9.0 release. I will edit/remove this note when the file path exists.